gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -305,24 +306,26 @@ public LeaseAttemptStatus tryAcquireLease(DagActionStore.DagAction dagAction, lo
           DagActionStore.DagAction updatedDagAction =
               adoptConsensusFlowExecutionId ? dagAction.updateFlowExecutionId(dbEventTimestamp.getTime()) : dagAction;
           log.debug("tryAcquireLease for [{}, is: {}, eventTimestamp: {}] - CASE 2: Same event, lease is valid",
-              updatedDagAction, isReminderEvent ? "reminder" : "original", dbCurrentTimestamp.getTime());
+              updatedDagAction, dagAction.isReminder ? "reminder" : "original", dbCurrentTimestamp.getTime());
           // Utilize db timestamp for reminder
-          return new LeaseAttemptStatus.LeasedToAnotherStatus(updatedDagAction, dbEventTimestamp.getTime(),
+          updatedDagAction.setEventTimeMillis(dbEventTimestamp.getTime());
```

Comment:
I see this being updated, but unless I missed something critical, it won't actually be durably persisted to the DB, will it?

`DagAction` is meant for sharing between the various distributed hosts.  in most any distributed sharing context, mutability is at best challenging, and at worst a tragic invitation to error.  therefore I DON'T SUGGEST to durably persist the event time.  immutability should be our standard technique.

where there's a need for additional local-to-each-host info, just compose around `DagAction`.  e.g.:
```
@Data
class ThisHostDagActionReflection {
  private final DagAction dagAction;
  private final long timestampMillis;
}
```
[yikes!  crappy name... sorry]

Reply:
I've really had a lot of trouble grasping the motivation behind this PR... but overall is the crux of it that you'd like `DagActionReminderScheduler` to be operate on its own personal extension of a `DagAction`, such as the example (`Reflection`) above?